src/gc/env/gcenv.base.h

```diff
@@ -7,10 +7,11 @@
 // Sets up basic environment for CLR GC
 //
 
-#define FEATURE_REDHAWK 1
-#define FEATURE_CONSERVATIVE_GC 1
+#ifdef _MSC_VER
+#include <intrin.h>
+#endif // _MSC_VER
 
-#define GCENV_INCLUDED
+#define FEATURE_REDHAWK 1
```

Comment:
The standalone GC does build with `FEATURE_REDHAWK` for now, as a stepping stone until the remainder of the Local GC work items are handled.